tests/js/spec/actionCreators/health.spec.jsx

```diff
@@ -0,0 +1,45 @@
+import {Client} from 'app/api';
+import {doHealthRequest} from 'app/actionCreators/health';
+
+describe('Health ActionCreator', function() {
+  const api = new Client();
+  const organization = TestStubs.Organization();
+  const project = TestStubs.Project();
+  let mock;
+
+  it('requests timeseries', function() {
+    mock = MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/health/graph/',
+    });
+    doHealthRequest(api, {
+      timeseries: true,
+      organization,
+      projects: [project],
+      environments: [],
+      tag: 'release',
+      topk: 5,
+    });
+
+    expect(mock).toHaveBeenCalled();
+
+    expect(mock.mock.calls[0][1].query).toMatchSnapshot();
```

Comment:
nit: I think these would be slightly better not as a snapshot (too easy to bulk overwrite these), but rather just asserting what the object should equal since it doesn't have that many properties and is *maybe* slightly easier to read inline.